beginner_source/blitz/tensor_tutorial.py

```diff
@@ -94,6 +105,30 @@
 # Tensor Operations
 # ~~~~~~~~~~~~~~~~~
 #
+# **Check for a tensor**
+# 
+# We can check whether an object in python is a tensor or not. Typically, is_tensor function checks 
+# and is_storage function checkes whether an object is stored as tensor object.
+x = [1, 2, 3, 4, 5]
+
+print(f"x is tensor: {torch.is_tensor(x)}\n")
+print(f"x is tensor: {torch.is_storage(x)}\n")
+
+y = torch.randn(1, 2, 3, 4, 5)
+print(f"x is tensor: {torch.is_tensor(y)}\n")
+print(f"x is tensor: {torch.is_storage(y)}\n")
+
+######################################################################
+# Here x is not a tensor
+# while y is a tensor but it is not stored.
+#
+# **Count Elements in a Tensor**
+#
+# To check the total number of elements in an input tensor object, the numerical element function can be used.
+print(f"Total number of elements in tensor y: {torch.numel(y)}\n")
+print(f"Total number of elements in 2D tensor: {torch.numel(x_data)}\n")
```

Comment:
Why is `x_data` suddenly here?

Reply:
Actually, I thought it would be good if we can reuse the previously defined data again but I think we should define x_data again so as to make it easy for the viewer. 
Relevant code is changed